docs/content/documentation/systems/Contact/index.md

```diff
@@ -1,6 +1,24 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # Contact System
+
+Finite element contact enforces constraints between surfaces in the mesh. Mechanical contact
+prevents penetration and develops contact forces. Thermal contact transfers heat between the
+surfaces. There are currently two systems to choose from for mechanical contact:
+Constraint and Dirac. Constraint based contact is recommended for two-dimensional problems
+and Dirac for three-dimensional problems. Constraint contact is more robust but due to the patch
```

Comment:
@bwspenc Is this comment about preferring Dirac contact for 3D problems still accurate?

Reply:
No, I would say that it's always better to use the Constraint system, and in fact, we hope to deprecate the Dirac system shortly.